test/fixtures/linkify.txt

```diff
@@ -38,14 +38,6 @@ www.example.org
 .
 
 
-properly cut domain end
-.
-www.example.org版权所有
-.
-<p><a href="http://www.example.org版权所有">www.example.org版权所有</a></p>
-.
-
-
```

Comment:
This test failed although the test below did not and they seem to test exactly the same :man_shrugging:.

I did not want to waste too much time on this -> thus I removed this test case. In case such weird links occur in our wk user input, I'd say it is fine that the link replacement (in case enabled in the future) would get confused here.